test/dotnet-test.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using Xunit;
+
+// Don't let the tests execute concurrently
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
```

Comment:
We run the same set of tests in net45 and netcoreapp (i.e. run tests on the test app from within dotnet-test.Tests). Would that cause this issue?